client/packages/common/src/ui/forms/JsonForms/api/hooks/document/useDocument.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from '@openmsupply-client/common';
+import { useDocumentApi } from '../utils/useDocumentApi';
+
+export const useDocument = (docName: string) => {
```

Comment:
:nit I would use `name` rather than `docName`. This is in the `useDocument` hook, so the document part is expected already 🤷  The `doc` doesn't add much; if needed, I'd use `documentName`